.chloggen/add_k8s_resources.yaml

```
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: k8s
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add hpa, resource_quota and replication_controller resource attributes
```

nit
```suggestion
note: Add horizontalpodautoscaler , resourcequota and replicationcontroller resource attributes
```

But TBH I'd prefer HPA since (I think) it fits under 'widely recognized' under k8s namespace

https://github.com/open-telemetry/semantic-conventions/blob/867ed9c88f916d587c154e0c7367d50511e11d6e/docs/general/naming.md?plain=1#L94-L101